packages/gcf-utils/package.json

```diff
@@ -40,7 +40,7 @@
     "get-stream": "^6.0.0",
     "into-stream": "^6.0.0",
     "pino": "^6.3.2",
-    "probot": "10.9.5",
+    "probot": "^11.0.1",
```

Comment:
Only worry - the floating `^` tends to bite us, and we used to directly pin.  Should we keep doing that?

Reply:
Oh, right, that makes sense! I'm fine keeping it at 11, then just manually upgrading when they move to 12.